content/md/products-super-power/how-to-manage-my-publications.md

```diff
@@ -45,77 +52,93 @@ To check which products are published:
 
 ## Unpublish a single product
 
-To unpublish one product:
-1.  Go to the published product grid by clicking on the `...` in the top right corner of the product grid screen
-1.  Click on the small cloud icon displayed on product rollover in the grid (see screenshot below)
+You can unpublish a product from the main product form or from the published product grid.
+
+From the published product grid:
+1.  Go to the published product grid (click on the `...` on the top right corner of the product grid screen)
+1.  Click on the small papersheet icon displayed on product rollover in the grid (see screenshot below)
+
+:::info
+You can also unpublish the product directly from the published product form.
+::
 
 ![Unpublished a product from the grid](../img/Products-PublishedProduct3.png)
 
-The product will be no longer published. The product will not be listed anymore in the published product grid.
+From the product grid:
+1.  Go to the product grid
+1.  Select the product to unpublish
+1.  Click on the `...` on the top right corner, 
+1.  Click on `Unpublish`
+
+The product will no longer be published and the product will not be listed anymore in the published product grid.
 
 ## Unpublish several products at once
 
 To unpublish several products at the same time:
-1.  Go to the Product grid
-1.  Click on the "..." in the top right corner of the screen
-1.  Select `Published products` in the dropdown
-1.  Tick the products in the grid
-1.  Click on the Bulk Actions button at the bottom of the screen, and select the unpublish operation
+1.  Go to the published product grid (click on the `...` in the top right corner of the product grid screen)
+1.  Select published products in the grid, the bulk action menu will appear at the bottom of the screen
+1.  Click on `Bulk action`
+1.  Then click on `Unpublish` button
 
 ![Bulk Actions on Publish products](../img/Products-PublishedProduct4.png)
 
 1.  Select the `Unpublish` operation:
-    *   If you are in charge of all selected products, follow the steps and all
-        products will be unpublished
-    *   If you are in charge of only part of the selected products: a message will appear to inform you that you can only unpublish the products for on which you have own rights. Then follow the next steps and only the products you own will be unpublished. The other products will not be unpublished
-    *   If you are in charge of none of the selected products: an error message will appear to inform you that you cannot unpublish the products because you do not have the rights to do so. Click on the “Back” button to exit the mass edit process
-
-    The combination of actions to unpublish / restore a version allows to push new information to a product version.
+    *   If you have the own permission on all the products you have selected, follow the steps and all the selected products will be unpublished
```

Comment:
"follow the steps": which steps? the steps below?
I think it would be nice to detail which steps are to be followed.

Reply:
Maybe adding an ending period (".") at the end of the sentence.